net/openvswitch/patches/0105-openssl-1.1.patch

```diff
@@ -0,0 +1,204 @@
+--- /dev/null
++++ b/lib/dhparams.c
```

Comment:
Is this change only about removing dependency on host openssl installation?

My local run of the `openssl dhparams` command cannot produce the same `dhp_2014` and other tables.  Can you share the implications?

Reply:
I've generated the parameters with openssl 1.1, and patched it so it would compile with openssl 1.0.  I will double check.  I might have done something wrong, since travis can't build it.